projects/plugins/crm/js/ZeroBSCRM.admin.invoicebuilder.js

```diff
@@ -1569,8 +1569,10 @@ function zbscrm_JS_calc_amount_due() {
 			// get
 			var v = jQuery( '.zbs-partial-value', ele ).text();
 
+			var label = jQuery( '.zlabel', ele ).text();
+
 			// detect +-
-			var multiplier = 1; // gets turned to -1 if negotive ()
+			var multiplier = 1; // gets turned to -1 if negative ()
 
 			// got -?
```

Comment:
It's unclear to me when the value amount would include a bracket, so leaving this for any unknown cases.